### ISSUE TITLE
[Request] Proxy Settings

### ISSUE THREAD

**Issue by Gymcap** [TachibanaGeneralLaboratories/download-navi#24]
Preface: I am _not_ asking you to host a proxy server.

I prefer FOSS software because I know it will generally respect my privacy better than proprietary software. The same seems to hold here with DLNavi.

I believe that allowing the user to filter their downloads through their proxy is just another step towards freedom in this sense.

I personally have a vpn subscription which provides a SOCKS5 proxy server for me to use. Many apps I own support this functionality to ensure the data sent through that particular app is handled safely, without the need to open my toggle my vpn every time. I think this functionality would compliment a FOSS download manager very well. Both for privacy, and freedom. 

TL;DR give users the option to download everything through a proxy

Example: Look at telegram proxy settings

**Comment by KrasnayaPloshchad** [comment]
[ADM](https://play.google.com/store/apps/details?id=com.dv.adm) is already implemented.

**Comment by scriptkitz** [comment]
please

**Comment by KaKi87** [comment]
Hello,
Any news on this ?
Thanks
